src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +145,56 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /*
+     * Returns if the player is banned or not (from the vanilla banning system)
+     *
+     * @return Banned state of player.
+     */
+    boolean isBanned();
```

Comment:
This already exists in [`User`](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/entity/player/User.java#L92)
Maybe some the methods should be in there too, i.e. `ban()`, `pardon()` and `getBanInfo()`